ISICbalance/ViewControllers/AccountViewController.swift

```diff
@@ -104,21 +108,22 @@ final class AccountViewController: BaseViewController, ValidateErrorPresentable
     func setupBindings() {
         usernameTextField <~> viewModel.username
         passwordTextField <~> viewModel.password
-        loginButton.reactive.isEnabled <~ viewModel.loginAction.isExecuting.negate()
+        loginButton.reactive.isEnabled <~ viewModel.actions.loginAction.isExecuting.negate()
         
-        viewModel.loginAction.errors
+        viewModel.actions.loginAction.errors
             .observe(on: UIScheduler())
             .observeValues { [weak self] _ in
-                self?.presentValidationError("You must fill out all fields.")
+                self?.presentValidationError(L10n.Validate.errorMessage)
             }
 
-        viewModel.loginAction.completed.producer.startWithValues { [weak self] in
-            self?.navigationController?.popViewController(animated: true)
+        viewModel.actions.loginAction.completed.producer.startWithValues { [weak self] in
```

Comment:
1) změnil bych naming `loginAction` na `login`. Máš už předtím namespace `actions`, takže je jasné, že to bude nějaká akce
2) `viewModel.actions.loginAction.completed.observeValues { [weak self] in ... }`, můžeš observovat přímo hodnoty padající ze `Signal`u